pkg/prometheus/resource_selector.go

```diff
@@ -1227,6 +1227,9 @@ func (rs *ResourceSelector) validateAzureSDConfigs(ctx context.Context, sc *moni
 
 func (rs *ResourceSelector) validateOpenStackSDConfigs(ctx context.Context, sc *monitoringv1alpha1.ScrapeConfig) error {
 	for i, config := range sc.Spec.OpenStackSDConfigs {
+		if strings.ToLower(config.Role) == "loadbalancer" && rs.version.LT(semver.MustParse("3.2.0")) {
```

Comment:
(nit) as @mviswanathsai mentioned somewhere else, let's add a const value to pkg/apis/monitoring/v1alpha1

Reply:
Fixed